### PR TITLE
Navigation Bar MD file + MD file edits

### PR DESCRIPTION
**Pull request by orinevares**
- created new folder and MD file for navigation bar component
- Removed UXPin as link in design with this on all MD files
- Changed title to How to use this on all MD files and language around it

### CODE REVIEW

File: components/navigation bar/README.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+4.	Robust
+* For all user interface components, the name and role can be programmatically determined (WCAG 4.1.2)
+
+## Design with This Component
+Download and use the B.C. government Design System library to help build prototypes and wireframes for your website or application.
+
+The library is constantly updated and currently available in the following tools:
+
+* Sketch: [https://sketch.cloud/s/Q0bkG]
```
[the remark this answers is not in the document]
Comment (orinevares) [reply components/navigation bar/README.md]: done

File: components/header/README.md [hunk 2]
```diff
@@ -89,7 +89,7 @@ Download and use the B.C. government Design System library to help build prototy
 The library is available in these design tools:
 
 * Sketch: [https://sketch.cloud/s/Q0bkG]
```
[the remark this answers is not in the document]
Comment (patricksimonian) [reply components/header/README.md]: looks like its still not been changed . change `* Sketch: [https://sketch.cloud/s/Q0bkG] ` to this `*[Sketch](https://sketch.cloud/s/Q0bkG)`

File: components/footer/README.md [hunk 3]
```diff
@@ -69,7 +69,6 @@ Colour Impairment Grayscale Keyboard Accessibility Screen Readers Tab Navigation
 Are you wire-framing or prototyping a site or app? You can download and use the B.C. government Design System library in one of the following design tools:
 
 *	Sketch: [https://sketch.cloud/s/Q0bkG]
```
[the remark this answers is not in the document]
Comment (orinevares) [reply components/footer/README.md]: re-done

File: components/header/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (orinevares) [reply components/header/README.md]: done

**Comment by orinevares** [comment]
It's ok to have the MD without the code and snippets because we want to use this as a reference for whoever does the code and have it available for them to reference.

File: components/header/README.md [hunk 4]
```diff
@@ -25,20 +38,7 @@ Good examples include:
...
-
-## Visual
-Header Visual:
-![Header](https://github.com/bcgov/design-system/blob/master/components/header/images/header.png?raw=true)
-
-## Use This For
-
-*	All public-facing online B.C. Government content and services.
-*	Websites or apps that do not require navigation built into the header.
+*Adapted from the UK Government’s [Service Naming Guide] (https://www.gov.uk/service-manual/design/naming-your-service)
```
Comment (patricksimonian) [inline components/header/README.md]: remove the white space between `[Service Naming Guide] (https://www.gov.uk/service-manual/design/naming-your-service)`

**Pull request merged**